src/pheval/prepare/create_spiked_vcf.py

```diff
@@ -206,7 +206,7 @@ def construct_variant_entry(self, proband_variant_data: ProbandCausativeVariant)
             "100",
             "PASS",
             proband_variant_data.info
-            if proband_variant_data.info is not None
+            if proband_variant_data.info != ""
```

Comment:
```suggestion
            if not proband_variant_data.info
```
This is a bit more pythonic, but does the same as the your suggestion